cypress/integration/book-page.test.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  TEST_BOOK,
+  TEST_COURSE,
+  TEST_COURSE_CODE,
+  TEST_DEPARTMENT,
+} from "cypress/support/constants";
+import { MAX_NUM_POSTS } from "@lib/helpers/constants";
+import { HttpMethod } from "@lib/http-method";
+
+describe("Course page", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should display info about the book", () => {
+    cy.visit("/book/" + TEST_BOOK.isbn);
+    cy.get('[test-id="Publisher"]').should(
+      "contain",
+      "Addison-Wesley Professional"
+    );
+    cy.get('[test-id="PublishedDate"]').should("contain", "2011");
+    cy.get('[test-id="ISBN"]').should("contain", TEST_BOOK.isbn);
+    cy.get('[test-id="PageCount"]').should("contain", "955");
+  });
+
+  it("should allow a user to navigate to the corresponding course page", () => {
+    cy.visit("/book/" + TEST_BOOK.isbn);
+    const button = cy.get('[test-id="CourseButton"]').first();
+    button.should(
+      "contain",
+      TEST_DEPARTMENT.abbreviation + " " + TEST_COURSE.code
+    );
+    button.click({ force: true });
+    cy.location("pathname").should("eq", "/course/" + TEST_COURSE_CODE);
+  });
+
+  it("should have a link to the google books page", () => {
+    cy.visit("/book/" + TEST_BOOK.isbn);
+    cy.get('[test-id="GoogleBookButton"]').should("have.attr", "href").and('include', `isbn:${TEST_BOOK.isbn}`);
+  });
+
+  it("should have a link to the book on the campus store", () => {
+    cy.visit("/book/" + TEST_BOOK.isbn);
+    cy.get('[test-id="CampusStoreButton"]').should(
+      "have.attr",
+      "href",
+      `https://campusstore.mcmaster.ca/cgi-mcm/ws/txsub.pl?wsDEPTG1=${TEST_DEPARTMENT.abbreviation}&wsCOURSEG1=${TEST_COURSE.code}`
+    );
```

Comment:
There is a second link to the campus store. If it isn't too much work, it would be nice to check for this as well.